CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changelog
 2.0.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add Python 3.9 base64.encodebytes() compatibility. Fix `#45 <https://github.com/collective/collective.jsonmigrator/issues/45>`_
```

Comment:
And please put your name in [CONTRIBUTORS.rst](https://github.com/collective/collective.jsonmigrator/blob/master/CONTRIBUTORS.rst)